lib/frrstr.h

```diff
@@ -166,6 +167,13 @@ int all_digit(const char *str);
  */
 char *frrstr_hex(char *buff, size_t bufsiz, const uint8_t *str, size_t num);
 
+
+/*
+ * Advance past a given char `skipc` in a string, while honoring quoting and
```

Comment:
I'm not sure that "while honoring quoting" explains the fact that we're completely skipping everything between the quotes. Or did I misunderstood the function?

Reply:
This is what it means ignore skipc between quotes.. I'll extend the comment.